tests/unittests/aggregation_tests.py

```
@@ -631,34 +650,63 @@ def test_values(self):
+            d._iter_values(F("predictions.detections").length())
+        )
+        self.assertListEqual(actual, itered_actual)
         self.assertListEqual(
-            d.values(F("predictions.detections").length()),
+            actual,
             [2, 3, 2, 0, 0],
         )
 
-        self.assertListEqual(
+        actual = d.values(
+            (F("predictions.detections.label") != None).if_else(
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)